jupyter_server_proxy/websocket.py

```diff
@@ -40,31 +40,21 @@ def pingable_ws_connect(
     request.headers = httputil.HTTPHeaders(request.headers)
     request = httpclient._RequestProxy(request, httpclient.HTTPRequest._DEFAULTS)
 
-    # for tornado 4.5.x compatibility
-    if version_info[0] == 4:
-        conn = PingableWSClientConnection(
-            io_loop=ioloop.IOLoop.current(),
-            compression_options={},
-            request=request,
-            on_message_callback=on_message_callback,
-            on_ping_callback=on_ping_callback,
-        )
```

Comment:
This is cleaned up now that we know we required tornado 5.0 via jupyter server 1.0.0+.